src/rust/ide/src/controller/graph/executed.rs

```diff
@@ -171,6 +164,32 @@ impl Handle {
         Ok(())
     }
 
+    /// Attempts to get the method pointer of the specified node.
+    ///
+    /// Fails if there's no information about target method pointer (e.g. because node value hasn't
+    /// been yet computed by the engine) or if method graph cannot be created (see
+    /// `graph_for_method` documentation).
+    pub fn node_method_pointer
+    (&self, node:double_representation::node::Id) -> FallibleResult<Rc<MethodPointer>> {
+        let registry   = self.execution_ctx.computed_value_info_registry();
+        let node_info  = registry.get(&node).ok_or_else(|| NotEvaluatedYet(node))?;
+        node_info.method_pointer.as_ref().cloned().ok_or_else(|| NoResolvedMethod(node).into())
+    }
+
+    /// Enter node by given ID.
+    ///
+    /// This will cause pushing a new stack frame to the execution context and changing the graph
+    /// controller to point to a new definition.
+    ///
+    /// Fails if there's no information about target method pointer (e.g. because node value hasn't
+    /// been yet computed by the engine) or if method graph cannot be created (see
+    /// `graph_for_method` documentation).
```

Comment:
similarly, this doc should be shared with `enter_method_pointer`, as some fail cases are now handled there.